### PR TITLE
Add online competition privacy notice to legal section

### LINKED ISSUE #7223: [COPY UPDATE] Update index.html - "Data Privacy"

### PR DESCRIPTION
**Pull request by deadlight**
## Done

Added competition privacy notice to the legal section

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8001/legal/data-privacy
- See the link to "Online competitions privacy notice" and click it
- Compare the page with the [copydoc](https://docs.google.com/document/d/1vy3Ta47YBWoKtvpbJyk01iYv3czAQMWz54G-UErA9zE/edit#)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

## Issue / Card

Fixes #7223

### CODE REVIEW

File: templates/legal/data-privacy/online-competitions.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+---
+
+<h4 class="p-muted-heading">Version - April 2020</h4>
+<hr style="margin-bottom: 2rem;" />
+
+# Privacy Notice – Online&nbsp;competitions
+
+This Privacy Notice tells you about the information collected from you when you enter into an online competition and agree to your personal data being stored by Canonical. In collecting this information, we are acting as a data controller and, by law, we are required to provide you with information about us, about why and how we use your data, and about the rights you have over your data. For more information, please see our privacy policy.
+
+# Who are we?
```
Comment (pmahnke) [inline templates/legal/data-privacy/online-competitions.md:16]: h2?

File: templates/legal/data-privacy/online-competitions.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+Your data may be shared with third parties as reasonably required for Canonical to process and store your data in accordance with this privacy notice or respond to your request.
+
+## Our legal basis to use your Personal information
+
+### Our legitimate business interests
+
+Our legal basis for using your personal information is that it is necessary for our legitimate interest in holding the online competition, for your participation in the online competition and in accordance with the terms and conditions between us.
+
+We may contact you from time to time with information about products and services that we offer (see <a href="https://ubuntu.com/legal/data-privacy/partner-portal#mkt-comms">Marketing Communications to Businesses</a>). Our legal basis for using your personal information to contact you is that it is necessary for our legitimate interest in keeping you informed of the products and services that we offer. You can opt out of marketing communications as set out below.
```
Comment (pmahnke) [inline templates/legal/data-privacy/online-competitions.md:43]: why have `https://ubuntu.com` ?

File: templates/legal/data-privacy/online-competitions.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+### Your consent
+
+If you enter into an online  competition, you will have the option to provide your consent to us sending you information about the products and services that we offer (see <a href="https://ubuntu.com/legal/data-privacy/partner-portal#mkt-comms">Email Marketing Communications</a>). You can withdraw your consent at any time.
```
Comment (pmahnke) [inline templates/legal/data-privacy/online-competitions.md:47]: why have `https://ubuntu.com` ?

File: templates/legal/data-privacy/online-competitions.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+## Changes to this Privacy Notice
+
+This Privacy Notice will be reviewed from time to time. If we decide to change this Privacy Notice we will post the changes here. However, if we intend to make material changes to the way we use your personal information we will notify you before doing so. Any personal information held will be governed by our most current notice.
+
+## Your right to complain
+
+We hope that we can resolve any query or concern you raise about our use of your information. You can contact us at any time. Any questions, comments and requests regarding this Privacy Notice or any other related matter are welcomed and should be addressed to <a href="mailto:dataprotection@canonical.com">dataprotection@canonical.com</a> or to the address below:
+
+Legal, Canonical
```
Comment (pmahnke) [inline templates/legal/data-privacy/online-competitions.md:104]: these addresses look inline, can you copy how we did this from another of these data privacy docs?

File: templates/legal/data-privacy/online-competitions.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+5th Floor, Blue Fin Building
+110 Southwark Street
+London, SE1 0SU
+United Kingdom
+
+Alternatively, you can use the relevant contact us form.
+
+If you have a complaint about our use of your information, you can contact the Information Commissioner’s Office via their website at <a href="https://ico.org.uk/make-a-complaint">ico.org.uk/make-a-complaint</a> or write to them at:
+
+Information Commissioner's Office
```
Comment (pmahnke) [inline templates/legal/data-privacy/online-competitions.md:114]: these addresses look inline, can you copy how we did this from another of these data privacy docs?

File: templates/legal/data-privacy/online-competitions.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+Legal, Canonical
+5th Floor, Blue Fin Building
+110 Southwark Street
+London, SE1 0SU
+United Kingdom
+
+Alternatively, you can use the relevant contact us form.
+
+If you have a complaint about our use of your information, you can contact the Information Commissioner’s Office via their website at <a href="https://ico.org.uk/make-a-complaint">ico.org.uk/make-a-complaint</a> or write to them at:
```
Comment (pmahnke) [inline templates/legal/data-privacy/online-competitions.md:112]: p-link--external ?

**Comment by deadlight** [comment]
@pmahnke Sorry, I pushed got distracted and forgot that I meant to fix those addresses before I proposed. Fixing now.

**Pull request merged**